packages/generator/src/generator.js

```diff
@@ -184,6 +187,9 @@ or disable the build step: \`generate({ build: false })\``)
     // Start generate process
     while (this.routes.length) {
       let n = 0
+      if (typeof failOnError === 'number' && errors.length >= failOnError) {
```

Comment:
I think would be best if we support this as an additional argument for type safety and better CLI ags definition (`--max-fails` or something similar)

Reply:
Thanks for your review.

I'm agree with your idea, so we can leave `--fail-on-error` option as it is.
How about `--max-errors` instead of `--max-fails` because  `--fail-on-error` means generations fail when error occurs.

Reply:
Yes, that's good wording 👍

Reply:
Thx! I'll get work on it this weekend👍

also fix https://github.com/nuxt/docs/pull/2135 too